frame/support/test/tests/construct_runtime_ui/undefined_genesis_config_part.stderr

```diff
@@ -1,78 +1,30 @@
 error: `Pallet` does not have #[pallet::genesis_config] defined, perhaps you should remove `Config` from construct_runtime?
-  --> $DIR/undefined_genesis_config_part.rs:5:1
+  --> tests/construct_runtime_ui/undefined_genesis_config_part.rs:5:1
```

Comment:
What causes it to go between `$DIR` and `tests/construct_runtime_ui`?

Reply:
I actually have no idea, I simply ran the tests without the test expectations, which would put the stderr files under a wip folder, and I then move those files to the `construct_runtime_ui` directory.